src/main/java/com/microsoft/lst_bench/client/SparkConnection.java

```diff
@@ -36,6 +36,11 @@ public void execute(String sqlText) {
     }
   }
 
+  @Override
+  public Object executeQuery(String sqlText) throws ClientException {
+    return session.sql(sqlText).collectAsList();
```

Comment:
Does this return similar `Object` as you are creating for `JDBCConnection`? If it is not (and currently we are just using the `JDBCConnection` to test the new dependent tasks), consider throwing an `UnsupportedOperationException` and creating a GH issue to address later.

Reply:
Done.